Bio/SCOP/__init__.py

```diff
@@ -467,31 +502,48 @@ def write_cla_sql(self, handle):
         cur = handle.cursor()
 
         cur.execute("DROP TABLE IF EXISTS cla")
-        cur.execute("CREATE TABLE cla (sunid INT, sid CHAR(8), pdbid CHAR(4),\
+        cur.execute(
+            "CREATE TABLE cla (sunid INT, sid CHAR(8), pdbid CHAR(4),\
         residues VARCHAR(50), sccs CHAR(10), cl INT, cf INT, sf INT, fa INT,\
-        dm INT, sp INT, px INT, PRIMARY KEY (sunid), INDEX (SID) )")
+        dm INT, sp INT, px INT, PRIMARY KEY (sunid), INDEX (SID) )"
+        )
```

Comment:
This is odd - I don't think we need the slashes inside the SQL command string - SQL should cope just fine with the command as one long string. In that case, Python implicit string concatenation would be neatest, so something like this:

```python
        cur.execute(
            "CREATE TABLE cla (sunid INT, sid CHAR(8), pdbid CHAR(4), "
            "residues VARCHAR(50), sccs CHAR(10), cl INT, cf INT, sf INT, fa INT, "
            "dm INT, sp INT, px INT, PRIMARY KEY (sunid), INDEX (SID))"
        )
```